filet.c

```diff
@@ -43,6 +44,7 @@ static struct termios g_old_termios;
 static int g_row;
 static int g_col;
 static bool g_needs_redraw;
+static volatile sig_atomic_t g_quit;
```

Comment:
```suggestion
static volatile sig_atomic_t g_quit = 0;
```
Instead of initializing it at the beginning of main.

Reply:
Static variables are always inited to 0 afaik, so neither are needed.

Reply:
Jep the C99 standard says that. The more you know :)
From compiler side it shouldn't make a difference, but it is better to read if initialized I think.

Reply:
Yep, if I rely on something to be initialized in a certain way, it's more clear if it's just explicit.